wolfcrypt/src/sp_int.c

```diff
@@ -77,6 +77,8 @@ This library provides single precision (SP) integer math functions.
  *      called again until complete.
  * WOLFSSL_SP_FAST_NCT_EXPTMOD  Enables the faster non-constant time modular
  *      exponentation implementation.
+ * WOLFSSL_SP_INT_DIGIT_ALIGN   Enable when unaligned access of sp_int_digit
```

Comment:
Should we turn this on for platforms where we know it should be on?  ARM for example.

Reply:
The report was for the Renesas SH4A chipset.
The chip has an unaligned move instruction but the compiler isn't using it.
Other files have specific defines for alignment but nothing is general except for WOLFSSL_GENERAL_ALIGNMENT which I assume means that it is faster to access not a fault.

I don't know of any chipset that requires this - I believe it will be compiler specific.